### PR TITLE
Add Network name

### PR DESCRIPTION
**Pull request by nerdlet**
Add network node for each sensor

Screenshots

Screenshot 2021-01-22 at 18 41 00

Screenshot 2021-01-22 at 18 40 45
Screenshot 2021-01-22 at 18 40 35

### CODE REVIEW

File: src/CNAME [hunk 1]
```diff
@@ -1 +1 @@
-wb.map.sensors.africa 
\ No newline at end of file
+//wb.map.sensors.africa
```
Comment (kilemensi) [inline src/CNAME]: Why are we commenting this out?

File: src/js/index.js [hunk 2]
```diff
@@ -598,49 +598,71 @@ function reloadMap(val) {
...
-	let textefin = "<table id='results' style='width:380px;'><tr><th class ='title'>" + translate.tr(lang, 'Sensor') + "</th><th class = 'title'>" + translate.tr(lang, titles[user_selected_value]) + "</th><th>Last date update</th></tr>";
+	function getNetwork(data) {
+		if (data === 24) {
+			return "AirQO"
+		} else if (data === 23) {
+			return "PurpleAir"
+		} else if (data === 22) {
+			return "OpenAQ"
+		} else {
+			return "SensorsAPI"
```
Comment (kilemensi) [inline src/js/index.js]: `
```suggestion
			return "sensors.AFRICA"
```

**Review by kilemensi — COMMENTED** [review]
🚀 @nerdlet ... Just the formatting of code and lets merge this.

File: src/CNAME (hunk 1 above)
Comment (nerdlet) [reply src/CNAME]: @kilemensi Sorry, forgot to uncomment it. The map is not showing with the CNAME link in it on local.

File: src/js/index.js [hunk 3]
```diff
@@ -604,7 +604,17 @@ function sensorNr(data) {
 		inner_pre = "(+) #";
 	}
 
-	openSidebar();
```
Comment (kilemensi) [inline src/js/index.js:607]: Don't need the `openSidebar()` ?

**Review by kilemensi — APPROVED** [review]
🚀

File: src/js/index.js (hunk 3 above)
Comment (nerdlet) [reply src/js/index.js:607]: @kilemensi Added it back

**Pull request merged**